Resources/Prototypes/_LostParadise/Entities/Structures/Wallmounts/newyear.yml

```
@@ -0,0 +1,87 @@
+- type: entity
+  parent: BaseSign
+  id: LPPNewYearSocksRed
+  name: socks
+  description: hanging socks on the wall, like you should put gifts there...
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)